### PR TITLE
[4.0] joomla.asset typos

### PR DESCRIPTION
**Pull request by brianteeman**
Simple pr to fix some typos

### CODE REVIEW

File: administrator/templates/atum/joomla.asset.json [hunk 1]
```diff
@@ -52,7 +52,7 @@
       "name": "template.active",
       "type": "script",
       "uri": "",
-      "description": "A dummy asset to allow extensions to use it as a dependency to active template",
+			"description": "A dummy asset to allow extensions to use it as a dependency to the active template",
```
Comment (wilsonge) [inline administrator/templates/atum/joomla.asset.json]: revert tabs here please and in the other file

Comment (brianteeman) [reply administrator/templates/atum/joomla.asset.json]: grrh

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 8ac571bcb50c55045664bb0c6d56c13aac8e6c8c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28484">issues.joomla.org/tracker/joomla-cms/28484</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
thanks

**Pull request merged**